### PR TITLE
Feature/e UI 3920 feature guard

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3920

### Change description ###

Guard the release 2 routes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/app/guards/work-allocation-guard.spec.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        expect(guard).toBeTruthy();
+    });
+
+    it('canActivate true', () => {
+        featureService.getActiveWAFeature.and.returnValue(of('WorkAllocationRelease2'))
+        const result = guard.canActivate();
+        result.subscribe(isActive => expect(isActive).toBeTruthy());
+    });
+
+    it('canActivate true', () => {
```
Comment (connorpgpmcelroy) [inline src/app/guards/work-allocation-guard.spec.ts:23]: Should this not be 'canActivate false'?

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]

File: src/app/guards/work-allocation-guard.spec.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/app/guards/work-allocation-guard.spec.ts]: yeah. changed it

File: src/app/guards/work-allocation-guard.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import { CanActivate } from '@angular/router';
+import { Observable, of } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
```
[the remark this answers is not in the document]
Comment (udaydenduluri33) [reply src/app/guards/work-allocation-guard.ts]: removed it

**Review by rayliangatos — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Pull request closed without merging**